src/Map/src/Bridge/Google/src/Option/FullscreenControlOptions.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
```

Comment:
This "Option" singular triggers me ... but this has no impact so it's just a comment ;)